packages/components/Molecules/Search/searchlayout.stories.js

```diff
@@ -0,0 +1,187 @@
+import RplSearchResults from './SearchResultsV2'
+import RplSearchResultsTable from './SearchResultsTable'
+import RplSearchResult from './SearchResult.vue'
+
+import { RplCol } from '@dpc-sdp/ripple-grid'
+import { RplCardProfile } from '@dpc-sdp/ripple-card'
+import { withKnobs, object, number } from '@storybook/addon-knobs/vue'
+
+export default {
+  title: 'Molecules/Search/SearchResults',
+  decorators: [withKnobs],
+  includeStories: ['withCards', 'withTable', 'withSearchResult']
+}
+
+const getItems = (item, count) => {
+  const items = []
+  for (let index = 0; index < count; index++) {
+    items.push(item)
+  }
+  return items
+}
+
+export const withSearchResult = () => ({
+  components: { RplSearchResults, RplCol, RplSearchResult },
+  template:
+    `
+    <rpl-search-results
+    :searchResults="searchResults"
+    :pager="pagination"
+    :count="20"
+    :perPage="10"
+  >
+    <template v-slot:results="scoped">
+      <rpl-col cols="full" v-for="(result, i) in scoped.searchResults" :key="i + '-result'">
+        <rpl-search-result
+          :title="result.title"
```

Comment:
Looks good!
![lg](https://media.makeameme.org/created/looks-good-to-5c78ba.jpg)